vcr/assets/assets/contexts/nuts.ldjson

```diff
@@ -34,8 +34,6 @@
 
         "id": "@id",
         "type": "@type",
-        "cred": "https://www.w3.org/2018/credentials#",
-        "nuts": "https://nuts.nl/credentials/v1#",
```

Comment:
does line 29 use this?

Reply:
defined at highest level. Seems to propagate.